_posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown

```diff
@@ -0,0 +1,179 @@
+---
```

Comment:
Overall thoughts:
I enjoyed reading your first article!  In general I think it would help me to understand more about the intentions of your blog, and the message you want to get across for an individual article. 

A few Questions:
What type of blog is this?  
What do you want people to get out of this article?

Thoughts:
- My impression is this is a general thoughts blog/ book review blog.
- In terms of grammar I think there may be an issue with commas (I mess this up all the time when I write, so i’m not sure.)
- I left a few comments on how I would reword things.
- Everything seems factually accurate.  When I joined Lyft I remember how paranoid everyone was of Uber spies (they were fine to me… maybe a little suspicious at first.). I made a joke oce about Uber spies, and everyone was deadpan like “No really this was a problem for us over the past year.”  I remember going to meetings where Travis specifically talked about his distrust of Didi and how he DID NOT want any talent from Didi.  He talked about corporate spies from Didi a lot and how he found people on both payrolls.  I do think the book is pretty spot on from the quotes you posted.
- I think you quoted the book a lot, and I wanted to see more of your interpretations on the situation and your perspective as a tech employee.  I think this article reads like a teaser for the book (which may be your intention).   
- I’m not completely about quotes you posted in relation to your opinion of him changing.  You acknowledge that he’s more than a douchey business person, but the quotes support him being just a douchy business person.